test/shell/exec-usergroup.sh

```diff
@@ -76,3 +76,6 @@ cleanup
 run_usergroup_test "exec-usergroup-${GROUP}.yaml" "nobody" "${GROUP}"
 run_usergroup_test "exec-usergroup-user.yaml" "nobody" ""
 run_usergroup_test "exec-usergroup-group-${GROUP}.yaml" "" "${GROUP}"
+
+# avoid race against rm command from the shell test wrapper
```

Comment:
What race is that?

Reply:
Not 100% certain it's a race, but I saw this spectacular failure: https://travis-ci.org/purpleidea/mgmt/jobs/647560853?utm_medium=notification&utm_source=github_status